src/ScriptTools.sol

```diff
@@ -0,0 +1,161 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+// Copyright (C) 2017 DappHub, LLC
+// Copyright (C) 2022 Dai Foundation
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+pragma solidity >=0.8.0;
+
+import { VmSafe } from "forge-std/Vm.sol";
+import { stdJson } from "forge-std/StdJson.sol";
+
+import { WardsAbstract } from "dss-interfaces/Interfaces.sol";
+
+/** 
+ * @title Script Tools
+ * @dev Contains opinionated tools used in scripts.
+ */
+library ScriptTools {
+
+    VmSafe private constant vm = VmSafe(address(uint160(uint256(keccak256("hevm cheat code")))));
```

Comment:
I thought we didn't need to use this cheat initialization anymore. And any reason to use this subset `VmSafe`? Just because we do not need functions outside of it?

Reply:
I'm not sure actually. I just copied the code from forge-std: https://github.com/foundry-rs/forge-std/blob/master/src/StdJson.sol

Reply:
Ok I understand the initialization now, we do not usually need to set it as we are in the Test environment, which inherits from Base.sol which already does it. Here we are doing a library which doesn't have anything to do with the Base.sol file, so all good.
Regarding the other one, I can see in the Base.sol both `Vm` and `VmSafe` are initialized for `ScriptBase`. It would be interesting to understand what one and other conceptually mean.

Reply:
Just confirmed in Foundry chat that it's just for safety. If you don't need cheatcodes that modify state then it's set to prevent accidental usage.

Reply:
Great, thanks!